source/math/Quaternion.ooc

```diff
@@ -243,6 +242,30 @@ Quaternion: cover {
 		result := [this w, this x, this y, this z]
 		result
 	}
+	toFloatTransform2D: func -> FloatTransform2D {
+		normalized := this normalized
+		nw := normalized w
+		nx := normalized x
+		ny := normalized y
+		nz := normalized z
+		
+		nwSquared := nw squared()
+		nxSquared := nx squared()
+		nySquared := ny squared()
+		nzSquared := nz squared()
```

Comment:
This could be written like this instead:
```ooc
		(nw, nx, ny, nz) := (normalized w, normalized x, normalized y, normalized z)
		
		(nwSquared, nxSquared, nySquared, nzSquared) := (nw squared(), nx squared(), ny squared(), nz squared())
```
but I'm not sure it's a good idea. Take it or leave it.